src/components/common/ScheduleLayout/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { StyledScheduleLayoutContainer } from './ScheduleLayout.styled';
+import { Props } from './ScheduleLayout.types';
+
+import { Chip } from '../Chip';
+import { FlexBox } from '../FlexBox';
+import { Header } from '../Header';
+import { IconButton } from '../IconButton';
+import { SpeechBubble } from '../SpeechBubble';
+import { Body3, Head3 } from '../Typography';
+
+export const ScheduleLayout = ({
+  categories,
+  meetingName,
+  meetingStartDate,
+  meetingEndDate,
+  dueDateTime,
+  children,
+}: Props) => {
+  return (
+    <StyledScheduleLayoutContainer>
+      <FlexBox flexDir="column" justifyContent="flex-start" width="100%">
+        <Header
+          left={<IconButton iconName="jjakkakText" size={130} iconColor="WH" />}
+          right={<IconButton iconName="user" size={28} iconColor="WH" />}
+        />
+        <FlexBox width="100%" alignItems="flex-start" gap={2} padding="10px 20px 0px 20px">
```

Comment:
```suggestion
        <FlexBox width="100%" alignItems="normal" gap={2} padding="10px 20px 0px 20px">
```

왼쪽 정렬을 의도하신 것이라면 `normal`로 작성해도 충분할 것 같아요! `align-items`의 실제 기본 값이 `normal`이어서 왼쪽부터 놓이는게 맞는데, 제가 `FlexBox`에는 `center`를 기본값으로 가지도록 작성해 둔게 좀 문제인 것 같습니다 ㅎㅎ;;

Reply:
왼쪽 정렬 의도 맞아요..! 수정했습니당! `c61c065`